gridt/src/app/movements/add-movement/add-movement.page.html

```diff
@@ -18,7 +18,7 @@
       <ion-row>
         <ion-col size-sm="6" offset-sm="3">
           <ion-item>
-            <ion-label position="floating">Movement name</ion-label>
+            <ion-label position="floating">Movement name*</ion-label>
```

Comment:
Why the asterisk?

Reply:
Just some extra user feedback to let them know that is a mandatory field, even before entering anything.